src/main/scala/org/hammerlab/guacamole/commands/GermlineStandardCaller.scala

```diff
@@ -51,8 +51,6 @@ object GermlineStandard {
 
       val readSet = Common.loadReadsFromArguments(args, sc, Read.InputFilters(mapped = true, nonDuplicate = true))
       readSet.mappedReads.persist()
-      Common.progress(
-        "Loaded %,d mapped non-duplicate reads into %,d partitions.".format(readSet.mappedReads.count, readSet.mappedReads.partitions.length))
```

Comment:
Gotcha, that seems borderline so either way works for me

On Thu, Jan 22, 2015 at 12:51 PM, Arun Ahuja <notifications@github.com>
wrote:

> In
> src/main/scala/org/hammerlab/guacamole/commands/GermlineStandardCaller.scala
> <https://github.com/hammerlab/guacamole/pull/276#discussion_r23394218>:
>
> > @@ -51,8 +51,6 @@ object GermlineStandard {
> >
> >        val readSet = Common.loadReadsFromArguments(args, sc, Read.InputFilters(mapped = true, nonDuplicate = true))
> >        readSet.mappedReads.persist()
> > -      Common.progress(
> > -        "Loaded %,d mapped non-duplicate reads into %,d partitions.".format(readSet.mappedReads.count, readSet.mappedReads.partitions.length))
>
> There is not that much of a speed-up in dropping , the count was around
> 30-40 seconds of time, so we can leave it in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/guacamole/pull/276/files#r23394218>.
>